lm_eval/tasks/mmmu/utils.py

```diff
@@ -13,7 +13,7 @@
 
 {}
 
-Answer with the option's letter from the given choices directly."""
+Answer with the option letter from the given choices directly."""
```

Comment:
I still see changes to the mmmu utils, IIRC we shouldn't need to change anything with the MMMU implementation?